src/blueprints/multilingual/templates/multilingual/editor.html

```diff
@@ -241,8 +241,21 @@ <h2 class="mb-0">
                  aria-disabled></div>
         </div>
         <div class="text-center d-inline-block w-100">
-            <div class="mt-4" id="errorMessage"></div>
-            <button class="btn button btn-primary mt-4" onclick="convert()">{{_('convert_button')}}</button>
+            <div class="mt-3" id="errorMessage"></div>
+            <button class="btn button btn-primary my-3" onclick="convert()">{{_('convert_button')}}</button>
+            <button class="btn button btn-primary my-3" onclick="runit()">{{_('execute_button')}}</button>
+        </div>
+        <div class="row">
+            <div class="execution-input col border border-dark mx-2" id="editor-execution-input-text"
+                 disabled aria-disabled>{{_('execution_input_text')}}</div>
+        </div>
+        <div class="row">
+            <div class="execution-input col border border-dark mx-2 mb-3" id="editor-execution-input"
+             disabled aria-disabled></div>
+        </div>
```

Comment:
The layout is not intuitive as it is. The distinction between program input and output should be made clearer. Maybe by putting these two side by side, or maybe by having a visible separation between the two.